Tests/Controller/LoginControllerTest.php

```diff
@@ -90,6 +91,20 @@ public function testLoginPage()
         $controller->connectAction($this->request);
     }
 
+    public function testLoginPageWithoutToken()
+    {
+        $this->mockAuthorizationCheck(new AuthenticationCredentialsNotFoundException());
```

Comment:
```suggestion
        $this->authorizationChecker
                ->method('isGranted')
                ->willThrowException(new AuthenticationCredentialsNotFoundException())
        ;
```
I think I would use this instead of changing the `mockAuthorizationCheck` method